docs/pages/demo/daterangepicker/BasicDateRangePicker.example.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
```

Comment:
`import React from 'react'` - actually is a weird workaround coming from interop between UMD and esm/cjs. We should not use default export from `React`

Reply:
We have a thread on this topic at https://github.com/mui-org/material-ui/pull/19802. Ok, it's not important, let's ignore it, it will be solved with the repository migration anyway.